src/plugins/intel_gpu/include/intel_gpu/plugin/graph.hpp

```diff
@@ -53,6 +53,10 @@ class Graph {
     std::shared_ptr<cldnn::engine> GetEngine() const { return getContextImpl(m_context)->GetEngine(); }
     int GetMaxDynamicBatchSize() const { return getConfig().max_dynamic_batch; }
     const std::map<std::string, cldnn::layout>& GetInputLayouts() const { return m_program->GetInputLayouts(); }
+    const InferenceEngine::InputsDataMap GetNetworkInputs() const { return m_program->GetNetworkInputs(); }
+    const InferenceEngine::OutputsDataMap GetNetworkOutputs() const { return m_program->GetNetworkOutputs(); }
+    std::map<std::string, std::pair<int64_t, int64_t>> GetInputDynBatchDims() { return m_program->m_input_batch_dim; }
```

Comment:
what is `std::pair<int64_t, int64_t>` ? Why not `ov::Dimension`?

Reply:
Because this is different functionality.
This is not an interval.
First value is **index** of batch dimension.
So, in my opinion, ov::Dimension will not cover this functionality properly.